packages/augur-ui/src/modules/common/selection.styles.less

```diff
@@ -27,30 +27,34 @@
       margin-right: auto;
     }
 
-    > svg > path {
-      fill: none;
-      stroke: @color-inactive-text;
+    > svg {
+      height: @size-6;
+      margin: 0 @size-1 0 @size-13;
+      width: @size-10;
+
+      > path {
+        fill: @color-primary-text;
+      }
     }
 
     &:hover {
       background: fade(@color-secondary-action, 50%);
       outline: none;
-
-      > svg > path {
-        stroke: @color-primary-text;
-      }
     }
   }
 
   > div:first-of-type {
-    background-color: @color-primary-text;
+    background-color: @color-secondary-action;
+    border: 1px solid @color-secondary-action-outline;
     border-radius: @border-radius-default;
+    box-shadow: 0px @size-4 @size-4 rgba(0, 0, 0, 0.25);
```

Comment:
we don't use rgba and we also don't want to have a black color like that, can we use something like `fade(@color-dark-grey, 25%);` instead here or even a darker value thats close to black? in general we always want to take from the color pallet we already have in `_colors.less`

Reply:
Sure. I have updated the box-shadow to ```0 @size-8 @size-8 @color-dark-grey;```

```fade(@color-dark-grey, 25%);``` looks not dark enough.